docs/API.Stream.CreateIterator.md

```diff
@@ -49,7 +49,6 @@ session-ttl | The time, in seconds, until the iteration session expires. The ite
 
 ## Notes ##
 
-* If the iteration process fails in the middle for any reason, you will have to restart the process from the beginning (that is, iterate again over all infotons that match the query).
```

Comment:
If the iteration process fails in the middle for any reason, **you might be able to resume with last token within the TTL window, otherwise you'll have to** restart the process from the beginning (that is, iterate again over all infotons that match the query).

Reply:
Not sure I understand this. What is the "last token within the TTL window"? Can't we simply say the last token we received? Also, what does "you might be able" mean? I thought the purpose of the feature was to enable this. What does failure/success depend on? Are we saying that we recommend doing a retry, and if it fails, start from the beginning?